### PR TITLE
Add documentation for super admin 

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Super admin will replace account owners. Added a brief explanation of the super admin role and instructions on how to make/revoke super admin privileges.

### CODE REVIEW

**Review by rsalevsky — DISMISSED** [review]
We should mention somewhere that super admin the equivalent to account owners and that adding someone to account owners will make them super admin and vise versa.

**Comment by staceysalamon-aiven** [comment]
> We should mention somewhere that super admin the equivalent to account owners and that adding someone to account owners will make them super admin and vise versa.


Will do. Do we also need to say that account owners as a team is being removed?

File: docs/platform/concepts/projects_accounts_access.rst [hunk 1]
```diff
@@ -11,32 +11,40 @@ Organizations and organizational units
...
-* ACLs - Can be set on all levels (organization, organizational unit, and project)
+* Access control lists (ACLs): Can be set on all levels (organization, organizational unit, and project)
 
   * ACLs for service plans are inherited, meaning all projects within an organization or organizational unit will have the same service plan.
 
-* Groups - User groups managed at the organization level and assigned to projects
+* Groups: Managed at the organization level and assigned to projects
 
-* Teams - Specific to a single organization or organizational unit and cannot be shared between them
+* Teams: Specific to a single organization or organizational unit and cannot be shared between them
```
Comment (rsalevsky) [inline docs/platform/concepts/projects_accounts_access.rst:26]: Shall we add here that this is deprecated?

Comment (staceysalamon-aiven) [reply docs/platform/concepts/projects_accounts_access.rst:26]: Or delete it completely?

**Review by rsalevsky — APPROVED** [review]

**Pull request merged**